tests/ocpp-server/README.md

```diff
@@ -0,0 +1,40 @@
+# OCPP2 Mock Server
+
+This project includes a mock Open Charge Point Protocol (OCPP) version 2.0.1 server implemented in Python.
+
+## Prerequisites
+
+This project requires Python 3.7+ and the following Python packages:
+
+- `websockets`
+- `ocpp`
+
+You can install these packages using pip:
+```
+pip install websockets ocpp
```

Comment:
Have you already implemented the python virtualenv with pipenv? Requiring to install globally python packages for development is not required nowadays.